source/replication.txt

```diff
@@ -104,30 +104,45 @@ Automatic Failover
 ------------------
 
 When a primary does not communicate with the other members of the set
-for more than 10 seconds, an eligible secondary will hold an election
+for more than the configured :rsconf:`electionTimeoutMillis` period 
+(10 seconds by default), an eligible secondary holds an election
 to elect itself the new primary. The first secondary to hold an
 election and receive a majority of the members' votes becomes primary.
 
 .. include:: /includes/fact-replica-set-protocolVersion1.rst
 
 .. include:: /images/replica-set-trigger-election.rst
 
-Although the timing varies, the failover process generally completes
-within a minute. For instance, it may take 10-30 seconds for the
-members of a :term:`replica set` to declare a :term:`primary`
-inaccessible (see :rsconf:`~settings.electionTimeoutMillis`). One of
-the remaining secondaries holds an :term:`election` to elect itself as
-a new primary. The election itself may take another 10-30 seconds.
+Lowering the :rsconf:`~settings.electionTimeoutMillis` 
+replication configuration option from the default ``10000`` (10 seconds)
+can result in faster automated failover at the cost of increased
+frequency of elections.
 
-.. versionchanged:: 3.2
+The replica set can continue to serve read queries 
+if such queries are configured to 
+:ref:`run on secondaries <replica-set-read-preference>` while the
+primary is offline. The replica set cannot process write operations 
+until the election completes successfully. 
 
-   Starting in MongoDB 3.2, with the :ref:`replication election
-   enhancements <3.2-rel-notes-rs-enhancements>`, MongoDB reduces
-   replica set failover time. See :ref:`replication election
-   enhancements <3.2-rel-notes-rs-enhancements>` for details.
+.. include:: /includes/fact-retryable-writes-failover-election.rst 
 
-See :ref:`replica-set-elections` and
-:ref:`replica-set-rollbacks` for more information.
+The median time in which a cluster may operate without a primary
```

Comment:
Same comment as in other file. But also, should we mention median time and time before we talk about doing things to reduce the timing?